.ci/scripts/coverity.sh

```diff
@@ -0,0 +1,414 @@
+#!/bin/bash -eE
+set -o pipefail
+
```

Comment:
Formatting not fixed - use code auto formatter in IDE

Reply:
all formatting fixed and 1 disabled.